src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -218,7 +218,7 @@ public string Title
         /// Title content of AppBar. TitleContent is added to Children automatically.
         /// If TitleContent is not TextLabel, then Title does not set title text of the TitleContent.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
why don't you throw exception when TitleContent is not a TextLabel?

Reply:
It's because developers may want to set other types of object to the TitleContent area.
e.g. SearchBar which has search button and TextEditor